src/components/Nav/Nav.scss

```
@@ -0,0 +1,41 @@
+.nav {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  border-bottom: 1px solid black;
+  padding: 20px 0px;
+
+  .logo {
+    .fa-instagram {
+      font-size: 30px;
+      margin-left: 100px;
+    }
+    .logo-name {
+      font-family: 'Lobster', cursive;
+      font-size: 30px;
+      margin-left: 10px;
+      border-left: 2px solid;
+      padding-left: 10px;
+      text-decoration: none;
+      color: black;
+    }
+  }
+
+  .search {
+    width: 200px;
+    height: 25px;
+    background-color: #f8f8f8;
+    border: 1px solid #d7dce3;
+    &::placeholder {
+      text-align: center;
+    }
+  }
+
+  .icons {
+    margin-right: 50px;
+    .fa-regular {
+      font-size: 30px;
+      margin: 0px 5px;
+    }
```

```suggestion
  .icons {
    margin-right: 50px;
    
    .fa-regular {
      font-size: 30px;
      margin: 0px 5px;
    }
```

nesting 시 가독성을 위해 선택자 사이 한칸씩 띄어주세요!